core/src/main/java/com/github/slugify/Slugify.java

```diff
@@ -27,27 +27,28 @@ public Slugify() {
 		loadReplacements(BUILTIN_REPLACEMENTS_FILENAME);
 	}
 
-	public Slugify withCustomReplacement(String from, String to) {
+	public Slugify withCustomReplacement(final String from, final String to) {
 		customReplacements.put(from, to);
 		return this;
 	}
 
-	public Slugify withCustomReplacements(Map<String, String> customReplacements) {
+	public Slugify withCustomReplacements(final Map<String, String> customReplacements) {
 		this.customReplacements.putAll(customReplacements);
 		return this;
 	}
 
-	public Slugify withUnderscoreSeparator(boolean underscoreSeparator) {
+	public Slugify withUnderscoreSeparator(final boolean underscoreSeparator) {
 		this.underscoreSeparator = underscoreSeparator;
 		return this;
 	}
 
-	public Slugify withLowerCase(boolean lowerCase) {
+	public Slugify withLowerCase(final boolean lowerCase) {
 		this.lowerCase = lowerCase;
 		return this;
 	}
 
-	public String slugify(String input) {
+	public String slugify(final String text) {
+	    String input = text;
```

Comment:
Is there a reason to set the method parameter to final and initialize a new object to be able to modify it? And: Pls use tabs instead of spaces.